CONTRIBUTING.rst

```diff
@@ -48,7 +48,9 @@ Release process
 ---------------
 
 The release is done automatically via GitHub actions when a new tag
-is pushed.
+is pushed. A new tag can be pushed with::
+
+    pipx run --spec="zest.releaser[recommended]>=6.22.1" fullrelease
```

Comment:
can we run this without `pipx` ?

My vision for a new release would have been:

* Create a new PR in which the version number, release notes and release dates are manually updated
* have the PR reviewed and approved and then merge the PR
* once the PR is merged create a tag with git and push the tag, or use GitHub to create a new release page https://github.com/twisted/ldaptor/releases/new

Also, I see that in this PR the version was updated to `rc2.dev0` is there any convention for that.
Why not use just `2.0.0.dev0` or `2.0.0.rc2`

In semantic versioning I see `2.0.0-rc1` or `2.0.0-dev` ... for a pre-release https://semver.org/#spec-item-9

Reply:
@adiroiban this is to distinguish between releases on pypi and releases on github